src/server/auth.js

```diff
@@ -2,17 +2,17 @@ import passport from 'passport';
 import { Strategy } from 'passport-local';
 import User from './models/user';
 
+const oid = '_id';
+
 export default () => {
   // Serialize sessions
-  passport.serializeUser(function(user, done) {
-    done(null, user.id);
-  });
+  passport.serializeUser((user, done) => done(null, user[oid]));
 
   // Deserialize sessions
-  passport.deserializeUser(function(id, done) {
+  passport.deserializeUser((id, done) => {
     User.findOne({
```

Comment:
there's a `findById`... i'm surprised this works when `_id` is an ObjectId. hmm...